src/commands/analytics/fetch-job-show.ts

```diff
@@ -58,10 +57,15 @@ export default class FetchJobShow extends ApiCommand {
         );
 
         if (opts.flags.download && res.record.state === 'completed') {
-            const objs: Array<TableObjectResponse> = await Promise.all(
-                res.record.page_urls.map(this.fetchAndOutputFile)
-            );
-            for (const obj of objs) {
+            /* eslint-disable no-await-in-loop */
```

Comment:
add a comment that explains why we are doing this serial and ignoring sound advice